api/config/config_test.json

```diff
@@ -26,7 +26,10 @@
     "projectNamePrefix": "aou-rw-test-",
     "exportBigQueryTable": "all-of-us-workbench-test-bd.billing_data.rw_test_firecloud_view",
     "retryCount": 2,
-    "bufferCapacity": 300,
+    "bufferCapacity": {
+      "registered": 300,
+      "controlled_test": 100
```

Comment:
I think we will generally want "controlled" but I'm calling this "controlled_test" to make it more clear that this is not real.

It may not be an important distinction.